crates/trigger/src/cli/stdio.rs

```diff
@@ -95,6 +103,39 @@ impl<F: RuntimeFactors, U> ExecutorHooks<F, U> for StdioLoggingExecutorHooks {
         configured_app: &spin_factors::ConfiguredApp<F>,
     ) -> anyhow::Result<()> {
         self.validate_follows(configured_app.app())?;
+
+        if self.refresh_log {
+            let sanitized_component_ids: Vec<String> = configured_app
+                .app()
+                .components()
+                .map(|c| sanitize_filename::sanitize(c.locked.id.clone()))
+                .collect();
+
+            for sanitized_component_id in sanitized_component_ids {
+                if let Some(stdout_log_path) = santized_log_path(
+                    self.log_dir.as_deref(),
+                    &sanitized_component_id,
+                    STDOUT_LOG_FILE_SUFFIX,
+                ) {
+                    if let Err(err) = std::fs::File::create(stdout_log_path) {
+                        //TODO: figure out if we want to return error here
```

Comment:
As you have it now, `create` _will_ fail if the logs dir doesn't already exist, and that shouldn't be considered an error or warning.  But you might want to move it to after the "create the log directory" bit so that it doesn't fail...